src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -100,34 +78,20 @@
 helps['acr login'] = """
     type: command
     short-summary: Log in to a container registry through Docker.
-    examples:
-        - name: Log in to a container registry
-          text: >
-            az acr login -n MyRegistry
 """
 
 helps['acr show-usage'] = """
     type: command
     short-summary: Get the quota usages for a container registry.
-    examples:
-        - name: Get the quota usages for a container registry.
-          text: >
-            az acr show-usage -n MyRegistry
 """
 
 helps['acr credential show'] = """
     type: command
     short-summary: Get the login credentials for a container registry.
     examples:
-        - name: Get the login credentials for a container registry.
-          text: >
-            az acr credential show -n MyRegistry
-        - name: Get the username used to log into a container registry.
-          text: >
-            az acr credential show -n MyRegistry --query username
-        - name: Get a password used to log into a container registry.
+        - name: Get the username and password used to log into a container registry.
           text: >
-            az acr credential show -n MyRegistry --query passwords[0].value
+            az acr credential show -n MyRegistry --query {user:username, password:passwords[0].value} --output table 
```

Comment:
Why specifically table format?

Reply:
I'm making the assumption that users printing login information are going to be copy/pasting it somewhere and this is the easiest format to do that from. If you think it's more useful to display as JSON (for piping) let me know.

Reply:
I was just thinking we don't include `--output table` since it doesn't *have* to be table format.
I don't mind though so up to you.